### PR TITLE
Fill OWNERS file in test-infra subdirectories

### PR DESCRIPTION
**Pull request by Ressetkk**
Add initial OWNERS file for test-infra subdirectories. May be subject of change in the future.

### CODE REVIEW

File: prow/scripts/OWNERS
Comment (dekiel) [inline prow/scripts/OWNERS:10]: Please add other groups to approvers and reviewers. This should not be responsibility of tooling team only.

File: prow/jobs/OWNERS
Comment (dekiel) [inline prow/jobs/OWNERS:10]: Do we have separate OWNERS file for each subdirectory? Tooling team should not be by default owner of any subdirectory.

**Review by dekiel — CHANGES_REQUESTED** [review]

File: prow/images/OWNERS
Comment (dekiel) [inline prow/images/OWNERS:10]: We should define separate OWNERS for some subdirectories here as well. Following are images for which we should not be approvers.
buildpack-java-node
buildpack-java
buildpack-node-chromium
buildpack-node

File: prow/jobs/OWNERS
Comment (Ressetkk) [reply prow/jobs/OWNERS:10]: tooling-team is the owner of the entire repository. This is just a precaution when one of the files that's been changed does not belong to anyone then tooling-team will decide what to do with that file. Besides every directory has its own ownership for a specific interest group regardless. This is subject of change and should be adjusted once the proposed approval flow will have a green light.

Comment (Ressetkk) [reply prow/jobs/OWNERS:10]: In the future it would be best to create an on-call group that will take care of those things.

File: prow/images/OWNERS
Comment (Ressetkk) [reply prow/images/OWNERS:10]: Tooling-team should be the approver of all images. It is related to the tooling and works directly on Prow. For the reviewer part the console team should review these images, that's true, but I would leave this until the approval process will have a green light

File: prow/jobs/OWNERS
Comment (dekiel) [reply prow/jobs/OWNERS:10]: How on call group relate to this change? What this on-call group will do?

File: prow/images/OWNERS
Comment (dekiel) [reply prow/images/OWNERS:10]: Don't agree with you. These are images for specific tests. Tooling team is not responsible for all tests and we don't have required knowledge from all technologies used in tests. At present and according to the proposal members of developers group have rights to approve changes here as well. This change should not increase our workload for the time of tests.

**Comment by Ressetkk** [comment]
@dekiel I've pushed latest changes to the OWNERS file including your concerns. I've also checked the usage of buildpack-java, java-node, node, node-chromium. Only build pack-node is used by prowjobs so I've filled the OWNERS file for this specific image only, as the rest is not used or is a dependency to other buildpack images. I'd leave the ownership to them for prow/images root owners and then sort out those images. I've created #4628 for this particular thing

File: prow/jobs/OWNERS
Comment (Ressetkk) [reply prow/jobs/OWNERS:10]: We can create a group of people who will be responsible for maintaining the prow jobs. Since it might not be the tooling-team only concern to take responsibility of the entire prow/jobs directory

**Review by dekiel — APPROVED** [review]

**Review by adamwalach — APPROVED** [review]

**Pull request merged**
